apiserver/pkg/model/model.go

```diff
@@ -114,6 +130,18 @@ func UpdateModel(ctx context.Context, c client.Client, input *generated.UpdateMo
 	model.Spec.Description = pointer.StringDeref(input.Description, model.Spec.Description)
 	model.Spec.Types = pointer.StringDeref(input.Types, model.Spec.Types)
 
+	if *input.ModelSource == common.ModelSourceModelscope {
```

Comment:
According to the requirements, the `modelScoure` does not support modification, but the `huggingFaceRepo`, `modelScopeRepo` and `revision` support modification.